crates/schema-impl/src/schemas_impl/component.rs

```diff
@@ -170,12 +171,27 @@ impl SchemasInner {
             .push(component_schemas.as_component_metadata(name));
     }
 
-    pub(crate) fn apply_modify_component(&mut self, name: String, new_public_value: bool) {
+    pub(crate) fn apply_modify_component(
```

Comment:
I think this commit conflicts a bit with https://github.com/restatedev/restate/pull/1361. Could we wait with merging this piece until the other PR is merged?